src/bilder/images/edxapp/group_data/all.py

```diff
@@ -75,18 +70,16 @@
         "rapid-response-xblock",
         "ol-openedx-canvas-integration",
         "ol-openedx-rapid-response-reports",
-        "git+https://github.com/arslanashraf7/super-csv@740b7c76cff72f0bbe6b1f3dde3a1086cf8e1cc3#egg=super-csv",  # noqa: E501
     ],
 }
 
 edx_plugins_removed = {
     "mitxonline": [
         "edx-name-affirmation",
-        "super-csv",
     ],
-    "xpro": ["super-csv"],
-    "mitx": ["super-csv"],
-    "mitx-staging": ["super-csv"],
+    "xpro": [],
```

Comment:
I removed the values from the list to just keep the code readability and ease, But maybe I should have removed the keys from the dictionary itself but I wasn't sure what practice we are following. Let me know.

Reply:
This looks great, thanks!